### PR TITLE
Use actions partial on plantings

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

File: app/views/plantings/_card.html.haml [hunk 1]
```diff
@@ -37,22 +37,4 @@
           %dd= planting.finish_predicted_at if planting.finish_predicted_at.present?
 
     %p= render 'plantings/progress', planting: planting, show_explanation: true
-
-    = link_to 'Details', planting_path(planting),
-      class: 'btn btn-default btn-xs'
-
-    - if can?(:edit, planting) && can?(:create, Harvest)
```
Comment (pozorvlak) [inline app/views/plantings/_card.html.haml]: The `can?(:create, Harvest)` check is missing from _actions.html.haml - is it unnecessary, or should it be ported across?

File: app/views/plantings/_card.html.haml [hunk 2]
```diff
@@ -37,22 +37,4 @@
           %dd= planting.finish_predicted_at if planting.finish_predicted_at.present?
 
     %p= render 'plantings/progress', planting: planting, show_explanation: true
-
-    = link_to 'Details', planting_path(planting),
```
Comment (pozorvlak) [inline app/views/plantings/_card.html.haml]: This isn't present in _actions.html.haml - should it be?

**Comment by pozorvlak** [comment]
@Br3nda: are the answers to my questions "no, this should not be present in _actions.html.haml" and "yes, it is unnecessary"? If so, let's merge it.

**Comment by Br3nda** [comment]
You caught two things i missed. I'll fix them up before we merge this.

**Comment by Br3nda** [comment]
Supersceed by #1550

**Pull request closed without merging**